src/DataTable/CheckboxFilter.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { useRef } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Form, FormLabel, Input, Badge,
+} from '..';
+
+function CheckboxFilter({
+  column: {
+    filterValue, setFilter, Header, checkboxFilters,
+  },
+}) {
+  // creates a unique label that does not change on re-render in case there are multiple checkbox filters in the dom
+  const ariaLabel = useRef(`checkbox-filter-label-${Math.floor(Math.random() * 100000)}`);
+  const inputText = `Search ${Header}`;
+  const checkedBoxes = filterValue || [];
+  const changeCheckbox = (name) => {
+    if (checkedBoxes.includes(name)) {
+      const newCheckedBoxes = checkedBoxes.filter((val) => val !== name);
+      return setFilter(newCheckedBoxes);
+    }
+    checkedBoxes.push(name);
+    return setFilter(checkedBoxes);
+  };
+  return (
+    <Form.Group role="group" aria-labelledby={ariaLabel.current}>
+      <FormLabel id={ariaLabel.current} className="pgn__checkbox-filter-label">{inputText}</FormLabel>
+      {checkboxFilters.map(({ name, number }) => (
+        <div key={name} className="pgn__checkbox-filter align-items-center mb-2">
+          <Input
+            type="checkbox"
+            checked={checkedBoxes.includes(name)}
+            onChange={() => { changeCheckbox(name); }}
+          />
+          <label htmlFor={name} className="mb-0">
+            {name}
+          </label>
```

Comment:
The input doesn't have an id and therefore not connected to the label (via for). It might also be simpler to use Form.Check. Even then though, we should be concerned about duplicate ids. 

Another alternative that would allow us to avoid needing an id would be to put the input inside the label.

Reply:
I tried input inside the label at one point but the style issues were a problem. I'm gonna try Form.Check and hopefully that will fix everything at once. :)